sklearn/linear_model/tests/test_base.py

```diff
@@ -318,9 +322,10 @@ def test_linear_regression_positive_vs_nonpositive():
     assert np.mean((reg.coef_ - regn.coef_) ** 2) > 1e-3
 
 
-def test_linear_regression_positive_vs_nonpositive_when_positive():
+def test_linear_regression_positive_vs_nonpositive_when_positive(global_random_seed):
     # Test LinearRegression fitted coefficients
     # when the problem is positive.
+    rng = check_random_state(global_random_seed)
```

Comment:
updated for consistency: `check_random_state` is used in some functions, but not all

Reply:
```suggestion
    rng = np.random.RandomState(global_random_seed)
```